No function or associated item named `schema_container` found for struct

When generating ABI for a contract that includes the use of borsh-serialized types, it requires manually importing `borsh::BorshSchema` into the scope, otherwise this error is reported:

```console
error[E0599]: no function or associated item named `schema_container` found for struct `Pair` in the current scope
  --> src/lib.rs:20:1
   |
7  | pub struct Pair(u32, u32);
   | --------------- function or associated item `schema_container` not found for this struct
...
20 | #[near_bindgen]
   | ^^^^^^^^^^^^^^^ function or associated item not found in `Pair`
   |
   = help: items from traits can only be used if the trait is in scope
   = note: this error originates in the attribute macro `near_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
1  | use near_sdk::borsh::BorshSchema;
   |

For more information about this error, try `rustc --explain E0599`.
error: could not compile `adder` due to previous error
```

--- a/near-sdk-macros/src/core_impl/abi/abi_generator.rs
+++ b/near-sdk-macros/src/core_impl/abi/abi_generator.rs
@@ -251,7 +251,7 @@ fn generate_schema(ty: &Type, serializer_type: &SerializerType) -> TokenStream2 
             gen.subschema_for::<#ty>()
         },
         SerializerType::Borsh => quote! {
-            <#ty>::schema_container()
+            <#ty as near_sdk::borsh::BorshSchema>::schema_container()
         },
     }
 }

